backend/src/models/Transaction.ts

```diff
@@ -0,0 +1,35 @@
+import { DataTypes } from "sequelize";
+import { sequelize } from "../config/db";
+
+const Transaction = sequelize.define("transaction", {
+  amount: {
+    type: DataTypes.INTEGER.UNSIGNED,
+    allowNull: false,
+  },
+  user_id: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  status: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  bank: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  gateway: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+});
+
+Transaction.sync({ alter: true })
```

Comment:
why always alter?

Reply:
since I was changing the fields constantly I wanted to see the changes in the DB, I removed it.